recipes/fortran-magic/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.7" %}
+
+package:
+    name: fortran-magic
+    version: {{ version }}
+
+source:
+    fn: fortran-magic-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/f/fortran-magic/fortran-magic-{{ version }}.tar.gz
+    md5: dc57d9c5daa0e16f6ce315045481b63e
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - ipython
```

Comment:
That would be over-specifying it I think (it is a metapackage). ```jupyter-notebook``` is the *actual* code installer (I think, though I will find out in the next few days).